debugcc.c

```diff
@@ -147,6 +147,21 @@ static bool leaf_enabled(struct debug_mux *mux, struct debug_mux *leaf)
 	return !val;
 }
 
+static void measure_mccc(const struct measure_clk *clk)
+{
+	struct debug_mux *gcc = clk->primary;
+	unsigned long val;
+
+	mux_enable(clk->primary, clk->mux);
+
+	/* MCCC is always on, just read the rate and return. */
+	val = 1000000000000ULL / readl(clk->leaf->base + clk->leaf->mux_reg);
```

Comment:
I'd probably suggest adding new hook to replace the `raw_count_full` calculation inside the existing `measure()`. This way we can support more `strange` clocks in future.

Maybe it would be better to strip the MCCC for now, merge the rest, and then handle the MCCC?

Reply:
1. not sure, the "standard" path does more enabling/disabling/reading/writing, where mccc just pokes the GCC mux and reads the value

2. it's in separate commits, can be easily knocked off if need be!

Reply:
@konradybcio You can find my proposal at https://github.com/andersson/debugcc/pull/15. This should allow getting the same result in a generic way.

Reply:
@konradybcio I did skip MCCC for SM8150 at https://github.com/andersson/debugcc/pull/14/files#diff-96f981c99793bf2059baf9ebf1212e8dd14d8b5de5ec8a4c92662bbc57b51f13R250 which has since been merged, woudl you mind patching that SoC up while adding support for MCCC in the rebasE?